src/btc/types.ts

```diff
@@ -27,20 +28,18 @@ export enum BtcMethod {
   SendMany = 'btc_sendmany',
 }
 
-/**
- * Supported Bitcoin account types.
- */
-export enum BtcAccountType {
-  P2wpkh = 'bip122:p2wpkh',
-}
-
 export const BtcP2wpkhAccountStruct = object({
-  ...BaseAccount,
+  ...KeyringAccountStruct.schema,
+
+  /**
+   * Account address.
+   */
+  address: BtcP2wpkhAddressStruct,
 
   /**
    * Account type.
    */
-  type: literal(`${BtcAccountType.P2wpkh}`),
+  type: enums([`${BtcAccountType.P2wpkh}`]),
```

Comment:
Out of curiosity, is there any reason to use `enums` over `literal` here? It seems, we still use `literal` for eth types.

Reply:
No, I was probably going to change something else. I'll revert to `literal`.